wordpress-mobile/gutenberg-mobile
# Move BlockHolder to Gutenberg as BlockListBlock in @wordpress/block-editor

Step 6 of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is the next step in our port of the BlockList to gutenberg.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/16223

### Testing Instructions
- Boot the app on iOS and Android
- Check that the block list works as expected and that the accessibility labels are stll ok.

Update release notes:
